### ISSUE TITLE
contrast: true screws up panel layout

### ISSUE THREAD

**Issue by VIPStephan** [hananils/kirby-colors#8]
I don’t know what it is but if a colors field has help text assigned and I’m adding `contrast: true` to the blueprint, it screws up the layout of the field if it’s used inside a structure field. The help text is squished to the side when it should appear below the field.

**Comment by nilshoerrmann** [comment]
Would you mind sharing a screenshot? I‘ll take a look at the issue next week.

**Comment by VIPStephan** [comment]
Here’s a screenshot. The colors field is used inside a structure field; when adding or editing an item it shows up like this. Hope that helps. Without contrast: true the help text shows up correctly below the field (as in the tab label field in the image).

**Comment by VIPStephan** [comment]
Here’s a screenshot. The colors field is used inside a structure field; when adding or editing an item it shows up like this. Hope that helps. Without contrast: true the help text shows up correctly below the field (as in the tab label field in the image).

![colors_contrast](https://user-images.githubusercontent.com/13770606/75090521-09ef7500-5564-11ea-8369-88f5bd0214e0.png)

**Comment by nilshoerrmann** [comment]
Thanks!

**Comment by nilshoerrmann** [comment]
May ask for another things: would you mind sharing the yaml definition in your blueprint for that structure field? Thanks!

**Comment by VIPStephan** [comment]
Here you go. Found out that it has something to do with how `.k-colors-field.shows-contrast` is set to `display: grid` where without that class it isn’t.
```
title: Home
options:
  status: false
tabs:
  content:
    sections:
      menutext:
        type: fields
        fields:
          menutext:
            extends: sections/menutext
      welcome:
        type: fields
        fields:
          w_title:
            type: headline
            label: Welcome Section
          welcome_text:
            type: textarea
            label: Welcome Message
            buttons:
              - bold
              - italic
          welcome_cta:
            type: text
            label: CTA Button Text
          separator2:
            type: line
      hero:
        type: fields
        fields:
          h_title:
            type: headline
            label: Hero Section
          hero_slider:
            type: structure
            label: Hero Slider
            required: true
            fields:
              case_study:
                type: pages
                label: Page
                multiple: false
                required: true
                query: site.find('work').children
                width: 1/3
              tab_label:
                type: text
                label: Tab Label
                help: The text that appears on the clickable tabs to select the slide.
                required: true
                width: 1/3
              color:
                label: Associated Color
                type: colors
                default: '#233747'
                help: The color will be displayed as background color in the welcome section at the top
                required: true
                width: 1/3
          separator3:
            type: line
…
…
```

**Comment by nilshoerrmann** [comment]
Thanks for the blueprint. I'll take a look at the problem over the next days.

**Comment by nilshoerrmann** [comment]
@VIPStephan Could you please check out the `develop` branch? I just pushed a few changes that should hopefully fix your issue, see f6239fc.
